lib/rex/powershell/command.rb

```diff
@@ -317,6 +317,13 @@ def self.cmd_psh_payload(pay, payload_arch, template_path, opts = {})
     end
 
     compressed_payload = compress_script(psh_payload, nil, opts)
+
+    if opts[:prepend_protections_bypass]
+      bypass_amsi = Rex::Powershell::PshMethods.bypass_powershell_protections
+      bypass_amsi.gsub!('$', '`$')
```

Comment:
always escaping `$` will likely break it when used in `cmd.exe`.

Reply:
I think this has been fixed by only escaping when the "exec_in_place" option is given